### PR TITLE
Feature/dashboard fixes

### LINKED ISSUE #92: Project editing dashboard broken
Original title: When updating a project, there are submission fields generated which prevent the updating of the project.
image

### PR DESCRIPTION
**Pull request by JackCrescend**
Expands submission editing to match the full ISubmission interface

Should fix #92

### CODE REVIEW

File: components/ProjectEditPage.tsx [hunk 1]
```diff
@@ -446,28 +465,41 @@ export default function ProjectEditPage({ doc }: IProps) {
...
-								value={submission.message}
-								onChange={(event) => updateMessage(index, event.currentTarget.value)}
-								placeholder="Message"
-								autoCapitalize="words"
-								className="border border-red-300 rounded-md px-1 ml-2 w-96"
-							/>
-						)
-						: (
+				<div className="grid grid-cols-submissionGrid gap-2 mt-4">
+					<span>Type:</span>
```
Comment (GoldElysium) [inline components/ProjectEditPage.tsx:469]: Use labels for a11y?

File: components/ProjectEditPage.tsx [hunk 2]
```diff
@@ -476,13 +508,37 @@ export default function ProjectEditPage({ doc }: IProps) {
...
 								className="border border-red-300 rounded-md px-1 ml-2 w-96"
 							/>
-						)
-				}
-				<TrashIcon className="w-6 h-6 ml-2 cursor-pointer" onClick={() => removeSubmissions(index)} />
+						</>
+					)
+						: ''}
+
+					<span>Message:</span>
```
Comment (GoldElysium) [inline components/ProjectEditPage.tsx:515]: Messages are links to txt files.

**Review by GoldElysium — CHANGES_REQUESTED** [review]

File: components/ProjectEditPage.tsx (hunk 2 above)
Comment (JackCrescend) [reply components/ProjectEditPage.tsx:515]: ? I'm not sure I understand, submission.message has always been raw text string. I could expand submission.src as an optional text url field, but the front-end doesn't have rendering for it yet.

File: components/ProjectEditPage.tsx (hunk 1 above)
Comment (JackCrescend) [reply components/ProjectEditPage.tsx]: Pushed a new commit with labels

File: components/ProjectEditPage.tsx (hunk 2 above)
Comment (GoldElysium) [reply components/ProjectEditPage.tsx]: Oh wait 🤦‍♂️ nvm. Ignore me being stupid

**Review by GoldElysium — COMMENTED** [review]
Looks good to me, if tested, you're allowed to merge.

**Review by GoldElysium — APPROVED** [review]

**Pull request merged**